vaccine_feed_ingest/runners/al/arcgis/fetch.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+import pathlib
+import sys
+
+# import arcgis ingestor
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+from ingestors import arcgis  # noqa: E402
+
+output_dir = sys.argv[1]
+print(f"Starting AL scraper with {output_dir=}")
+
+
+query_url = """https://services7.arcgis.com/4RQmZZ0yaZkGR1zy/arcgis/rest/services/DriveThruWalkIn2_Public/FeatureServer/0/query"""  # noqa: E501
```

Comment:
Since they have multiple URLs we'll want to scrape, would we want to set a pattern of either having an array of URLs or of naming more descriptively?